.pre-commit-config.yaml

```diff
@@ -91,7 +91,7 @@ repos:
           - manual
         language: python
         additional_dependencies:
-          - clang-format==14.0.6
+          - clang-format==16.0.6
```

Comment:
No it was required to resolve the messed formatting I posted about above. I didn't pull 2.4 into `main` because [thats currently broken](https://github.com/mixxxdj/mixxx/pull/11839#issuecomment-1694210108) IIUC. I just figured I'd just update it here and the merge conflict will be easy to resolve later. The Alternative would be to cherry-pick #11889 into `main`.

Reply:
Hmm, my local test worked fine, but if we changed deps this does look weird. Since #11839 has already been merged I'd rather take care of updating the vcpkg env now rather than risking that changes in 2.4 pile up and become a pain to resolve later on.

Reply:
https://github.com/mixxxdj/mixxx/pull/11891

Reply:
I agree, I just don't like messing with the buildenv as that's not my area of expertise. Thank you for taking care.